doc/classes/Sprite.xml

```diff
@@ -68,10 +68,13 @@
 			The region of the atlas texture to display. [member region_enabled] must be [code]true[/code].
 		</member>
 		<member name="shininess" type="float" setter="set_shininess" getter="get_shininess" default="1.0">
+			Strength of the specular light effect of this [AnimatedSprite].
```

Comment:
Should there be `Sprite` instead of `AnimatedSprite`?

Reply:
Yes, copy-paste went too far on this one.